src/index.html

```diff
@@ -1,12 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+      content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet"
+      href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__day calendar__day--num--1"></div>
+      <div class="calendar__day calendar__day--num--2"></div>
+      <div class="calendar__day calendar__day--num--3"></div>
+      <div class="calendar__day calendar__day--num--4"></div>
+      <div class="calendar__day calendar__day--num--5"></div>
+      <div class="calendar__day calendar__day--num--6"></div>
+      <div class="calendar__day calendar__day--num--7"></div>
+      <div class="calendar__day calendar__day--num--8"></div>
+      <div class="calendar__day calendar__day--num--9"></div>
+      <div class="calendar__day calendar__day--num--10"></div>
+      <div class="calendar__day calendar__day--num--11"></div>
+      <div class="calendar__day calendar__day--num--12"></div>
+      <div class="calendar__day calendar__day--num--13"></div>
+      <div class="calendar__day calendar__day--num--14"></div>
+      <div class="calendar__day calendar__day--num--15"></div>
+      <div class="calendar__day calendar__day--num--16"></div>
+      <div class="calendar__day calendar__day--num--17"></div>
+      <div class="calendar__day calendar__day--num--18"></div>
+      <div class="calendar__day calendar__day--num--19"></div>
+      <div class="calendar__day calendar__day--num--20"></div>
+      <div class="calendar__day calendar__day--num--21"></div>
+      <div class="calendar__day calendar__day--num--22"></div>
+      <div class="calendar__day calendar__day--num--23"></div>
+      <div class="calendar__day calendar__day--num--24"></div>
+      <div class="calendar__day calendar__day--num--25"></div>
+      <div class="calendar__day calendar__day--num--26"></div>
+      <div class="calendar__day calendar__day--num--27"></div>
+      <div class="calendar__day calendar__day--num--28"></div>
+      <div class="calendar__day calendar__day--num--29"></div>
+      <div class="calendar__day calendar__day--num--30"></div>
+      <div class="calendar__day calendar__day--num--31"></div>
+    </section>
   </body>
+
 </html>
```

Comment:
```suggestion
  </body>
</html>
```

Reply:
for the next tasks: there shouldn’t be empty line between child and parent element